proj/proj.mk

```diff
@@ -81,6 +81,11 @@ ifdef SKIP_TFLM
 export DEFINES    += SKIP_TFLM
 endif
 
+# Need a slower baudrate when communicating with Serv
+ifdef SERV
+export UART_SPEED = 115200
+endif
+
```

Comment:
Can we make this be close to the default definition of `UART_SPEED` -- either move the def above down here, or move this up there, so that everything dealing with `UART_SPEED` is together.